crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -767,7 +775,7 @@ mod tests {
             let block1 = random_block(1, Some(genesis.hash), None, Some(0));
             insert_blocks(env.db.as_ref(), [&genesis, &block1].into_iter());
 
-            let _ = spawn_consensus_engine(consensus_engine);
+            let _ = spawn_consensus_engine(consensus_engine).await;
```

Comment:
the consensus engine is a future that shouldn't exit when ignored. to fix the warning you prob can keep the receiver returned from spawn and drop it at the end of the tests

Reply:
this awaits on the oneshot response though
which does not seem to receive anything

imo it is reasonable to await the receiver return value.